src/features/security/_mixins.scss

```diff
@@ -5,7 +5,7 @@
     align-items: center;
     margin: 0;
     padding: 2px 8px;
-    border-radius: $bdl-border-radius-large-size;
+    border-radius: 2px + 16px / 2; // rounded sides: vert padding + half the line height of content
```

Comment:
Is the math needed here or should we just set the actual value?

Reply:
@patrickpaul preferred showing why the value was that number

Reply:
Actually, decided to change it to just a hardcoded number and explanation